code/datums/diseases/advance/symptoms/damage_converter.dm

```diff
@@ -42,6 +42,10 @@ Bonus
 
 		var/list/parts = H.get_damaged_organs(1,1) //1,1 because it needs inputs.
```

Comment:
The problem is that you would have to then fix that for every single thing that calls that function. Which there are a lot of. We can either sacrifice a few extra cycles doing this, or remake that function and all it's calls. I suppose for the sake of future use it will have to be done though. Is there a refactor function in dream maker, or can you point me to a byond IDE with one?